package.json

```diff
@@ -87,18 +87,19 @@
     "packages/*"
   ],
   "resolutions": {
-    "gobbledygook": "https://github.com/mozilla-fxa/gobbledygook.git#354042684056e57ca77f036989e907707a36cff2",
-    "tap/typescript": "^4.5.2",
+    "@grpc/grpc-js": "~1.6.0",
     "@nestjs/cli/typescript": "^4.5.2",
     "@types/node": "^16.11.3",
-    "http-proxy": "^1.18.1",
     "browserid-crypto": "https://github.com/mozilla-fxa/browserid-crypto.git#5979544d13eeb15a02d0b9a6a7a08a698d54d37d",
-    "fbjs/isomorphic-fetch": "^3.0.0",
     "eslint-plugin-import": "^2.25.2",
-    "minimist": "^1.2.6",
-    "url-parse": "^1.5.8",
+    "fbjs/isomorphic-fetch": "^3.0.0",
+    "gobbledygook": "https://github.com/mozilla-fxa/gobbledygook.git#354042684056e57ca77f036989e907707a36cff2",
     "google-gax": "2.30.5",
-    "@grpc/grpc-js": "~1.6.0"
+    "http-proxy": "^1.18.1",
```

Comment:
Is there a reason for all the line changes? It looks like just a single resolution for moment was added, but it's kind of hard to see this.

Reply:
I `sort`'d over the lines.